docs: update install guide (#460)
nitrictech/docs: src/pages/guides/getting-started/installation.mdx

## Installing the Nitric CLI

<CodeGroup>

```bash {{ title: 'macOS', label: 'nitric is available via <a href="https://brew.sh/">homebrew</a>' }}
brew install nitrictech/tap/nitric
```

```bash {{ title: 'Windows', label: 'nitric is available via <a href="https://scoop.sh/">scoop</a>' }}
scoop bucket add nitric https://github.com/nitrictech/scoop-bucket.git
scoop install nitric
```

> ```bash {{ title: 'Linux', label: 'nitric can be downloaded as a scripted install via curl.' }}

Reviewer: Not sure about the labels.
![image](https://github.com/nitrictech/docs/assets/13415171/46f3342b-c1f8-4ccc-ab3c-bcb4ef706d14)

The text description is a bit superfluous.

Author: Agree, it should be fairly obvious. I added them so it mapped one to one to the old version of doc, but happy to remove.

curl https://nitric.io/install | bash
```

> ```bash {{ title: 'Brew', label: 'nitric is available via <a href="https://brew.sh/">homebrew</a>' }}

Reviewer 2: Does brew need its own tab given it's provided under MacOS?

Reviewer: This is a good point, worth thinking about as brew can also be used on Linux as well if users choose.

Author: ^ yeah I added it for linux users. But then I guess they have the install script too. I can move it as a part of the linux one?

brew install nitrictech/tap/nitric
```

</CodeGroup>

### Manual Install

> You can download pre-compiled binaries from the [releases](https://github.com/nitrictech/cli/releases) page and copy them into your desired location

Reviewer 2: ```suggestion
Alternatively, you can download pre-compiled binaries from the [releases](https://github.com/nitrictech/cli/releases) page and install them manually.
```
